project page.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
This is an example inline comment. 🙂 

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. 😉 

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.